src/scripts/main.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('tbody');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+function buttonDisadled() {
+  const rowCount = document.querySelectorAll('tr');
+  const columnCount = rowCount[0].children;
+
+  if (rowCount.length <= 2) {
+    removeRow.disabled = true;
+  } else {
+    removeRow.disabled = false;
+  }
```

Comment:
```suggestion
    removeRow.disabled = rowCount.length <= 2;
```
no need to use `if` block to assign a boolean value